lib/helpers.js

```diff
@@ -18,4 +18,25 @@ function fixCaps (originalCaps) {
   return caps;
 }
 
-export { fixCaps };
+function isDictionary (obj) {
+    if (!obj) return false;
+    if (Array.isArray(obj)) return false;
+    if (obj.constructor !== Object) return false;
+    return true;
+}
+
+function logObjectVerbose (obj) {
```

Comment:
do we like this format better than what we could get for free with util.inspect?